metricbeat/docker-compose.yml

```diff
@@ -1,81 +1,84 @@
-version: '2.1'
+version: '2'
 services:
   beat:
-    build: ${PWD}/.
+    build: .
     environment:
       - TEST_ENVIRONMENT=false
     working_dir: /go/src/github.com/elastic/beats/metricbeat
     volumes:
-      - ${PWD}/..:/go/src/github.com/elastic/beats/
+      - ./..:/go/src/github.com/elastic/beats/
       # This is required for the docker module tests
       - /var/run/docker.sock:/var/run/docker.sock
     command: make
     env_file:
-      - ${PWD}/module/aerospike/_meta/env
-      - ${PWD}/module/apache/_meta/env
-      - ${PWD}/module/ceph/_meta/env
-      - ${PWD}/module/couchbase/_meta/env
-      - ${PWD}/module/dropwizard/_meta/env
-      - ${PWD}/module/elasticsearch/_meta/env
-      - ${PWD}/module/haproxy/_meta/env
-      - ${PWD}/module/http/_meta/env
-      - ${PWD}/module/jolokia/_meta/env
-      - ${PWD}/module/kafka/_meta/env
-      - ${PWD}/module/kibana/_meta/env
-      - ${PWD}/module/kubernetes/_meta/env
-      - ${PWD}/module/memcached/_meta/env
-      - ${PWD}/module/mongodb/_meta/env
-      - ${PWD}/module/mysql/_meta/env
-      - ${PWD}/module/nginx/_meta/env
-      - ${PWD}/module/php_fpm/_meta/env
-      - ${PWD}/module/postgresql/_meta/env
-      - ${PWD}/module/prometheus/_meta/env
-      - ${PWD}/module/rabbitmq/_meta/env
-      - ${PWD}/module/redis/_meta/env
-      - ${PWD}/module/zookeeper/_meta/env
+      - ./module/aerospike/_meta/env
+      - ./module/apache/_meta/env
+      - ./module/ceph/_meta/env
+      - ./module/couchbase/_meta/env
+      - ./module/dropwizard/_meta/env
+      - ./module/elasticsearch/_meta/env
+      - ./module/haproxy/_meta/env
+      - ./module/http/_meta/env
+      - ./module/jolokia/_meta/env
+      - ./module/kafka/_meta/env
+      - ./module/kibana/_meta/env
+      - ./module/kubernetes/_meta/env
+      - ./module/memcached/_meta/env
+      - ./module/mongodb/_meta/env
+      - ./module/mysql/_meta/env
+      - ./module/nginx/_meta/env
+      - ./module/php_fpm/_meta/env
+      - ./module/postgresql/_meta/env
+      - ./module/prometheus/_meta/env
+      - ./module/rabbitmq/_meta/env
+      - ./module/redis/_meta/env
+      - ./module/zookeeper/_meta/env
 
   # Modules
   aerospike:
-    build: ${PWD}/module/aerospike/_meta
+    build: ./module/aerospike/_meta
```

Comment:
what I like from the global compose file is that it's handy to keep your scenario clean. In case of error or cancellation, I can do `docker-compose stop` and kill all containers. If we split this across modules I may need to go over all subfolders